<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>691</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>220</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabWidgetPage1">
          <attribute name="title">
           <string>Manual</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="horizontalSpacing">
            <number>9</number>
           </property>
           <property name="verticalSpacing">
            <number>14</number>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Seleccionar área de Interés:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="tbSelectZone">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Marcar Triángulo de Ward:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="tbSelectTWard">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Marcar Compresivo Principal:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QToolButton" name="tbSelectCP">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Determinar diagnóstico:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QToolButton" name="tbDiagnosticate">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabWidgetPage2">
          <attribute name="title">
           <string>Automático</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
         <property name="autoScrollMargin">
          <number>10</number>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Imágenes</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionAbrir_imagen"/>
    <addaction name="actionGuardar_resultados"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuFiltros">
    <property name="title">
     <string>Filtros</string>
    </property>
    <widget class="QMenu" name="menuDeteccion_de_bordes">
     <property name="title">
      <string>Deteccion de bordes</string>
     </property>
     <widget class="QMenu" name="menuDireccionales">
      <property name="title">
       <string>Direccionales</string>
      </property>
      <addaction name="actionEste"/>
      <addaction name="actionOeste"/>
      <addaction name="actionNorte"/>
      <addaction name="actionSur"/>
      <addaction name="actionSuroeste"/>
      <addaction name="actionSureste"/>
      <addaction name="actionNoroeste"/>
      <addaction name="actionNoreste"/>
     </widget>
     <widget class="QMenu" name="menuBasadas_en_gradiente">
      <property name="title">
       <string>Basadas en gradiente</string>
      </property>
      <addaction name="actionRoberts"/>
      <addaction name="actionSobel"/>
      <addaction name="actionPrewitt"/>
      <addaction name="actionFrei_Chen"/>
     </widget>
     <addaction name="menuDireccionales"/>
     <addaction name="menuBasadas_en_gradiente"/>
    </widget>
    <widget class="QMenu" name="menuRealce">
     <property name="title">
      <string>Realce</string>
     </property>
     <addaction name="actionPaso_Alto"/>
    </widget>
    <widget class="QMenu" name="menuOtros">
     <property name="title">
      <string>Suavizado</string>
     </property>
     <addaction name="actionPaso_Bajo"/>
     <addaction name="actionGaussiano"/>
    </widget>
    <addaction name="menuRealce"/>
    <addaction name="menuOtros"/>
    <addaction name="menuDeteccion_de_bordes"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <widget class="QMenu" name="menuHistograma">
     <property name="title">
      <string>Histograma</string>
     </property>
     <addaction name="actionUmbral"/>
     <addaction name="actionNormalizacion"/>
     <addaction name="actionEcualizacion"/>
    </widget>
    <addaction name="menuHistograma"/>
   </widget>
   <widget class="QMenu" name="menuTransformaciones">
    <property name="title">
     <string>Transformaciones</string>
    </property>
    <addaction name="actionBinarizar_imagen"/>
    <addaction name="separator"/>
    <addaction name="actionDilatacion"/>
    <addaction name="actionErosion"/>
    <addaction name="actionErosi_n_condicional"/>
    <addaction name="actionDilataci_n_condicional"/>
    <addaction name="separator"/>
    <addaction name="actionApertura"/>
    <addaction name="actionCierre"/>
    <addaction name="separator"/>
    <addaction name="actionEliminar_huecos"/>
    <addaction name="actionEsqueletizacion"/>
    <addaction name="actionEliminar_ruido"/>
    <addaction name="separator"/>
    <addaction name="actionBuscar_Ward"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuFiltros"/>
   <addaction name="menuTransformaciones"/>
   <addaction name="menuHerramientas"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir_imagen">
   <property name="text">
    <string>Abrir imagen</string>
   </property>
   <property name="toolTip">
    <string>Abrir imagen para analizar.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGuardar_resultados">
   <property name="text">
    <string>Guardar resultados</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPaso_Alto">
   <property name="text">
    <string>Paso Alto</string>
   </property>
  </action>
  <action name="actionPaso_Bajo">
   <property name="text">
    <string>Paso Bajo</string>
   </property>
  </action>
  <action name="actionEste">
   <property name="text">
    <string>Este</string>
   </property>
  </action>
  <action name="actionOeste">
   <property name="text">
    <string>Oeste</string>
   </property>
  </action>
  <action name="actionNorte">
   <property name="text">
    <string>Norte</string>
   </property>
  </action>
  <action name="actionSur">
   <property name="text">
    <string>Sur</string>
   </property>
  </action>
  <action name="actionSuroeste">
   <property name="text">
    <string>Suroeste</string>
   </property>
  </action>
  <action name="actionSureste">
   <property name="text">
    <string>Sureste</string>
   </property>
  </action>
  <action name="actionNoroeste">
   <property name="text">
    <string>Noroeste</string>
   </property>
  </action>
  <action name="actionNoreste">
   <property name="text">
    <string>Noreste</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionFrei_Chen">
   <property name="text">
    <string>Frei-Chen</string>
   </property>
  </action>
  <action name="actionCompleto">
   <property name="text">
    <string>Completo</string>
   </property>
  </action>
  <action name="actionGaussiano">
   <property name="text">
    <string>Gaussiano</string>
   </property>
  </action>
  <action name="actionDilatacion">
   <property name="text">
    <string>Dilatación</string>
   </property>
  </action>
  <action name="actionErosion">
   <property name="text">
    <string>Erosión</string>
   </property>
  </action>
  <action name="actionErosi_n_condicional">
   <property name="text">
    <string>Erosión condicional</string>
   </property>
  </action>
  <action name="actionUmbralado">
   <property name="text">
    <string>Umbralado</string>
   </property>
  </action>
  <action name="actionNormalizaci_n">
   <property name="text">
    <string>Normalización</string>
   </property>
  </action>
  <action name="actionUmbral">
   <property name="text">
    <string>Umbralado</string>
   </property>
  </action>
  <action name="actionNormalizacion">
   <property name="text">
    <string>Normalización</string>
   </property>
  </action>
  <action name="actionEcualizacion">
   <property name="text">
    <string>Ecualización</string>
   </property>
  </action>
  <action name="actionEsqueletizacion">
   <property name="text">
    <string>Esqueletizacion</string>
   </property>
  </action>
  <action name="actionBuscar_Ward">
   <property name="text">
    <string>Buscar Ward</string>
   </property>
  </action>
  <action name="actionEliminar_ruido">
   <property name="text">
    <string>Eliminar ruido</string>
   </property>
  </action>
  <action name="actionEliminar_huecos">
   <property name="text">
    <string>Eliminar huecos</string>
   </property>
  </action>
  <action name="actionDilataci_n_condicional">
   <property name="text">
    <string>Dilatación condicional</string>
   </property>
  </action>
  <action name="actionApertura">
   <property name="text">
    <string>Apertura</string>
   </property>
  </action>
  <action name="actionCierre">
   <property name="text">
    <string>Cierre</string>
   </property>
  </action>
  <action name="actionUmbralizar_automaticamente">
   <property name="text">
    <string>Umbralizar automaticamente</string>
   </property>
  </action>
  <action name="actionBinarizar_imagen">
   <property name="text">
    <string>Binarizar imagen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tbSelectZone</tabstop>
  <tabstop>tbSelectTWard</tabstop>
  <tabstop>tbSelectCP</tabstop>
  <tabstop>tbDiagnosticate</tabstop>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
